### ISSUE TITLE
Nothing happens

### ISSUE THREAD

**Issue by naxIO** [iiska/middleman-webp#17]
I'm trying to get this extension to work, but no success so far.

Gemfile:
`gem 'middleman-webp'`

config.rb

```
configure :build do
   activate :minify_css
   activate :minify_javascript
   activate :gzip
   activate :minify_html
   activate :webp
   activate :imageoptim do |options|
...
```

prequesites are installed via homebrew.

webp 1.3.0 is already installed and up-to-date.

when I run middleman build everythings works fine but the images I have are not touched at all by webp. Imageoptim does its job afterwards.

I'm using ruby `3.0.0p0 (2020-12-25 revision 95aff21468) [arm64-darwin20]`

**Comment by iiska** [comment]
This gem has been in some serious need for maintenance love, but just now merged #16 fixes issues related to Ruby 3.0 support.

These fixes were just released as [gem version 1.0.0](https://rubygems.org/gems/middleman-webp/versions/1.0.0) (I decided to jump major version, since support for Rubies <= 2.4 was dropped same time)

Maybe you could try again with that, and report if you still had any problems?

**Comment by naxIO** [comment]
Thank you! Worked with middleman-webp (1.0.2)!
